### DISCUSSION TITLE
Is RustDesk reporting my collected information according to fdroid?

### DISCUSSION THREAD

**Discussion by Raj2032** [rustdesk/rustdesk#2807] [General]
According to fdroid's RustDesk https://f-droid.org/en/packages/com.carriez.flutter_hbb/

I can see that it has an antifeature listed which is "This app tracks and reports your activity":

![image](https://user-images.githubusercontent.com/29293830/211818072-30cefeac-1fca-42f5-8220-8dacd74efc50.png)

What sort of activity is the app reporting?

**Comment by bigretromike** [comment]
I would assume because it by default connect to not YOURS remote server ? (but I did not test this, just a good guess)
In contrast VNC does not use default repeater server, you need to configure that, and RustDesk use the "official" server to be able to connect and/or generate ID number that you can use to connect.

**Comment by Raj2032** [comment]
Ah right that makes sense, thanks mate
